updater.js

```diff
@@ -0,0 +1,233 @@
+require("dotenv").config();
+const util = require("util");
+const exec = util.promisify(require("child_process").exec);
+const versionNumberRegexp = /([vV])?([0-9]{1,2})\.([0-9]{1,2})(?:\.([0-9]{1,2}))?(?:\.([0-9]{1,2}))?[\n|\s]?/g;
+const wallets = require("./wallets");
+const writeFile = util.promisify(require("fs").writeFile);
+const manifest = require("./package");
+
+class Version {
+  constructor(versionRegexpResult) {
+    this.prefix = versionRegexpResult[1] || "";
```

Comment:
There is a prettier in this project, all your code should be prettified. The command:
`"format": "prettier --trailing-comma es5 --single-quote --print-width 120 --write \"src/**/*.js\"",`
might need to be changed to include your code.